bookwyrm/templates/author/author.html

```diff
@@ -22,42 +22,74 @@ <h1 class="title">{{ author.name }}</h1>
+                        </dd>
+                    {% endfor %}
                 </div>
                 {% endif %}
                 {% if author.born %}
-                <div class="is-flex">
-                    <dt class="mr-1">{% trans "Born:" %}</dt>
-                    <dd itemprop="aliases">{{ author.born|naturalday }}</dd>
+                <div class="is-flex mb-2">
+                    <dt class="has-text-weight-bold mr-1">{% trans "Born:" %}</dt>
+                    <dd itemprop="birthDate">{{ author.born|naturalday }}</dd>
                 </div>
                 {% endif %}
                 {% if author.died %}
-                <div class="is-flex">
-                    <dt class="mr-1">{% trans "Died:" %}</dt>
-                    <dd itemprop="aliases">{{ author.died|naturalday }}</dd>
+                <div class="is-flex mb-2">
+                    <dt class="has-text-weight-bold mr-1">{% trans "Died:" %}</dt>
+                    <dd itemprop="deathDate">{{ author.died|naturalday }}</dd>
                 </div>
                 {% endif %}
             </dl>
 
             {% if author.wikipedia_link %}
-            <p><a href="{{ author.wikipedia_link }}" rel=”noopener” target="_blank">{% trans "Wikipedia" %}</a></p>
+            <p class="mb-1">
+                <a itemprop="sameAs" href="{{ author.wikipedia_link }}" rel=”noopener” target="_blank">
+                    {% trans "Wikipedia" %}
+                </a>
+            </p>
             {% endif %}
+
             {% if author.openlibrary_key %}
-            <p class="mb-0">
-                <a href="https://openlibrary.org/authors/{{ author.openlibrary_key }}" target="_blank" rel="noopener">{% trans "View on OpenLibrary" %}</a>
+            <p class="mb-1">
+                <a itemprop="sameAs" href="https://openlibrary.org/authors/{{ author.openlibrary_key }}" target="_blank" rel="noopener">
+                    {% trans "View on OpenLibrary" %}
+                </a>
             </p>
             {% endif %}
+            
             {% if author.inventaire_id %}
-            <p class="mb-0">
-                <a href="https://inventaire.io/entity/{{ author.inventaire_id }}" target="_blank" rel="noopener">{% trans "View on Inventaire" %}</a>
+            <p class="mb-1">
+                <a itemprop="sameAs" href="https://inventaire.io/entity/{{ author.inventaire_id }}" target="_blank" rel="noopener">
+                    {% trans "View on Inventaire" %}
+                </a>
+            </p>
+            {% endif %}
+
+            {% if author.librarything_key %}
+            <p class="mb-1">
+                <a itemprop="sameAs" href="https://www.librarything.com/author/{{ author.librarything_key }}" target="_blank" rel="noopener">
+                    {% trans "View on LibraryThing" %}
+                </a>
+            </p>
+            {% endif %}
+            
+            {% if author.goodreads_key %}
+            <p class="mb-1">
```

Comment:
Mmm… Not a blocker, just a comment:

I don’t have a strong opinion about `mb-x` or `mt-x` but if a `mb-x` happens on the last item, it will add an extra vertical space at the bottom of the surrounding `.box`. That’s why I had proposed `mt-x` on the successive `<p>`: there would already be the `dl` sibling before the first `p.mt-x`, allowing to style its following `p` consistently without adding extra space.

Reply:
I added the `mb-x` to all the elements for consistency. I don't know which elements will be present in the box. It may be just a Wikipedia link, or just two definitions, or the whole enchilada. What I could do, is apply `my-2` to all the line elements (`p` and `dl > div`) and removing the `.box` vertical padding (with `py-0`). The only problem would be the `dl` keeping margins to be collapsed. I'll try it out.

Reply:
![image](https://user-images.githubusercontent.com/1781070/119368889-d72ab400-bcb3-11eb-98e7-5d40a658b688.png)

Here's what I got with b72580d

Reply:
I was feeling like `mt-x` on all `p` would work for whatever number of `p` without any other change, but the `dl` might be empty, OK.

Reply:
The styling is out of the scope of this PR anyway. All’s good for me.